Otherworldly Pines/Assets/Scripts/FallingDamage.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FallingDamage : MonoBehaviour
+{
+    public float damageImpulseThreshold = 5000f;
+    public float highestDamagingImpulse = 10000f;
+    public float lowestDamageDealt = 10f;
+    public float HighestDamageDealt = 30f;
+
+    private Vector3 objectVelocity;
+
+    // Update is called once per frame
+    void Update()
+    {
+        objectVelocity = this.GetComponent<Rigidbody2D>().velocity;
+
+        if (objectVelocity.y > damageImpulseThreshold)
+        {
+
+        }
+    }
```

Comment:
All this can be deleted

Reply:
Oh yeah, it's the debris when I tried a different approach using velocity to calculate damage.